src/vectorchord/getting-started/overview.md

```diff
@@ -82,9 +82,10 @@ SELECT * FROM items ORDER BY embedding <-> '[3,1,2]' LIMIT 5;
 For more usage, please read:
 
 * [Indexing](../usage/indexing)
-* [Indexing with MaxSim Operators](../usage/indexing-with-maxsim-operators)
+* [Search](../usage/search)
+* [Monitoring](../usage/monitoring)
 * [Performance Tuning](../usage/performance-tuning)
-* [Advanced Features](../usage/advanced-features)
+* [Advanced Features](../advanced-features/cold-start-optimization)
```

Comment:
Create an `advanced-features` page. `use-cases` page is done that way. Or inline it.